lib/components/narrative/line-itin/itin-summary.js

```diff
@@ -102,27 +106,53 @@ export default class ItinerarySummary extends Component {
       <Container onClick={this._onSummaryClicked}>
         <Details>
           {/* Travel time in hrs/mins */}
-          <Header>{coreUtils.time.formatDuration(itinerary.duration)}</Header>
+          <Header>
+            <FormattedDuration
+              duration={itinerary.duration}
+              id='common.time.tripDurationFormat'
+              zeroHoursId='common.time.tripDurationFormatZeroHours'
+            />
+          </Header>
 
           {/* Duration as time range */}
           <Detail>
-            {coreUtils.time.formatTime(itinerary.startTime, timeOptions)} - {coreUtils.time.formatTime(itinerary.endTime, timeOptions)}
+            <FormattedTime
+              startTime={itinerary.startTime}
+              endTime={itinerary.endTime}
+              timeFormat={timeFormat}
+            />
           </Detail>
 
           {/* Fare / Calories */}
           <Detail>
             {minTotalFare > 0 && <span>
-              {centsToString(minTotalFare)}
-              {minTotalFare !== maxTotalFare && <span> - {centsToString(maxTotalFare)}</span>}
+              <FormattedNumber
+                value={minTotalFare / 100}
+                style='currency'
+                currencyDisplay='narrowSymbol'
+                currency={currency}
+              />
+              {minTotalFare !== maxTotalFare &&
+                <span> -
+                  <FormattedNumber
+                    value={maxTotalFare / 100}
+                    style='currency'
+                    currencyDisplay='narrowSymbol'
+                    currency={currency}
+                  />
+                </span>}
               <span> &bull; </span>
             </span>}
-            {Math.round(caloriesBurned)} Cals
+            {Math.round(caloriesBurned)} <FormattedMessage id='components.ItinerarySummary.calories' />
```

Comment:
I found multiple spellings for the Calories abbreviation:
- This component (used in mobile view of itinerary results) displays "Cals" (as in "100 Cals").
- The tabbed itinerary summary (in desktop view) displays "Cal" (as in "100 Cal").
I am in favor of merging the two and say, in English, "100 Cal".

Reply:
I have changed the components I touched to "Cal" as of https://github.com/opentripplanner/otp-react-redux/pull/408/commits/4bbbe49e5ea70baf91a3f834ed42f5a2ec93bc19